internal/assets/token_transfer_test.go

```diff
@@ -91,8 +95,10 @@ func TestMintTokensSuccess(t *testing.T) {
 	mti.On("MintTokens", context.Background(), mock.Anything, "F1", &mint.TokenTransfer).Return(nil)
 	mth.On("SubmitNewTransaction", context.Background(), "ns1", fftypes.TransactionTypeTokenTransfer).Return(fftypes.NewUUID(), nil)
 	mdi.On("InsertOperation", context.Background(), mock.Anything).Return(nil)
-
+	mmi.On("IsMetricsEnabled").Return(true)
```

Comment:
Just as an idea for the future, might be worth adding `enableMetrics` to the `newTestAssets` call to avoid this boilerplate on so many tests

Reply:
Good idea. There is now a `newTestAssets` and `newTestAssetsWithMetrics`